cmd/client/Dockerfile

```diff
@@ -54,7 +54,7 @@ EXPOSE 42777
 EXPOSE 32111
 
 COPY --from=builder $SRC_PATH/drand-client /usr/local/bin/drand-client
-COPY --from=builder $SRC_PATH/cmd/drand-client/entrypoint.sh /usr/local/bin/entrypoint.sh
+COPY --from=builder $SRC_PATH/internal/drand-client/entrypoint.sh /usr/local/bin/entrypoint.sh
```

Comment:
How comes there are scripts in `internal` that are required for the client Dockerfile?